tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -90,5 +90,163 @@ static void TestLifetime(OwnedBuffer<byte> owned)
                 var span = copyStoredForLater.Span;
             });
         }
+
```

Comment:
Ah yeah missed that, great. However unlikely it is to catch the GC issue in these kinds of tests. But good.